### DISCUSSION TITLE
[Android 14] Error in Widget (Glance App Widget)

### DISCUSSION THREAD

**Discussion by rk-sTEk** [TechbeeAT/jtxBoard#931] [General]
Hi,

after update to Android 14 the widget don't work - App runs.
The shown error is Error in GlanceAppWidget (see pic).
![Uploading Screenshot_20231005-104029.png…]()

**Comment by patrickunterwegs** [comment]
Hey, thank you for the issue report, this is solved in jtx Board 2.6.0 which was released yesterday!

**Comment by rk-sTEk** [comment]
Made a bug report. Have the newest version from f-Droid and the problem continues. Even with new added Widget for test.

**Comment by patrickunterwegs** [comment]
Hi @rk-sTEk 

As I said, the problem is fixed in jtx Board 2.6.0, I have no influence on when f-Droid takes the new version.

**Reply by rk-sTEk** [reply]
Ah, okay. Didn't knew that the stores publish program versions delayed.

**Reply by patrickunterwegs** [reply]
Yes, f-Droid is taking the source from published release versions. It depends on their cycles when they take up the latest version, it could be the same day or several days after...
For other stores (like Google Play) there's more control.
